### DISCUSSION TITLE
Issue when building with webpack 5

### DISCUSSION THREAD

**Discussion by nikeshmhr** [aws/aws-sdk-js-v3#3564] [General]
Confirm by changing [ ] to [x] below:
- [x] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/welcome.html) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/)
- [x] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js) for answers

**Describe the question**
I'm getting the following issue while using `aws-amplify` and `aws-amplify-react` with webpack 5
![Screenshot from 2021-08-24 17-24-53](https://user-images.githubusercontent.com/7429948/130610226-afe4f68e-6b1d-44da-9d5c-a6459f4b171b.png)

It works when I downgrade my webpack version to 4.

Here's the repo for the code https://github.com/nikeshmhr/amplify-test

**Comment by ajredniwja** [comment]
Transferring to the correct repo.

**Comment by ajredniwja** [comment]
Related to https://github.com/aws/aws-sdk-js-v3/issues/2564
